components/raftstore/src/store/config.rs

```diff
@@ -137,6 +140,14 @@ pub struct Config {
     #[online_config(skip)]
     pub snap_apply_batch_size: ReadableSize,
 
+    #[doc(hidden)]
+    #[online_config(skip)]
+    pub region_worker_tick_interval: ReadableDuration,
+
+    #[doc(hidden)]
+    #[online_config(skip)]
+    pub stale_peer_check_tick: usize,
```

Comment:
I think it's ambiguous. There is also `peer_stale_state_check_interval` in the same struct. Hard to tell the difference.

Reply:
The previous author use constant `STALE_PEER_CHECK_TICK`, so I use `stale_peer_check_tick `

Reply:
That's because the constant is defined in region worker, it has context.